### PR TITLE
Updated benchmarks

### PR DESCRIPTION
**Pull request by mcollina**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -149,13 +149,13 @@ __Method:__: `autocannon -c 100 -d 40 -p 10 localhost:3000` * 2, taking the seco
 
 | Framework          | Version                    | Router?      |  Requests/sec |
 | :----------------- | :------------------------- | :----------: | ------------: |
-| hapi               | 18.1.0                     | &#10003;     | 29,998        |
-| Express            | 4.16.4                     | &#10003;     | 38,510        |
-| Restify            | 8.0.0                      | &#10003;     | 39,331        |
-| Koa                | 2.7.0                      | &#10007;     | 50,933        |
-| **Fastify**        | **2.0.0**                  | **&#10003;** | **76,835**    |
+| Express            | 4.17.1                     | &#10003;     | 15,978        |
```
Comment (Eomm) [inline README.md:152]: It is less than half of the previous value, should we update the benchmark endpoint?

Comment (mcollina) [reply README.md:152]: what do you mean?

Comment (Eomm) [reply README.md:152]: I think it is odd the value from 38,510 to 15,978 Requests/sec

I'm asking if there is someone who knows if there is some new express configuration that requires to update the "hello world" endpoint - or maybe they have just a bug in the latest version

Comment (mcollina) [reply README.md:152]: I did not dig into why.

**Pull request merged**